ggezchain/chain.json

```diff
@@ -0,0 +1,188 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "GGEZ1 Chain",
+  "chain_type": "cosmos",
+  "chain_id": "ggezchain",
+  "website": "https://ggez.one/",
+  "pretty_name": "GGEZ1 Chain",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "ggez",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md